webui/src/lib/components/auth/layout.tsx

```diff
@@ -23,6 +25,12 @@ export const AuthLayout = () => {
     return (
         <Container fluid="xl">
             <Row className="mt-5">
+                <div>
```

Comment:
How can users dismiss this?  At the very least let's add an `id` to allow user scripts to do this, but ideally we'd have a persistent "X" button.

Reply:
Are you suggesting to use a [flash-message](https://www.npmjs.com/package/react-native-flash-message/v/0.1.18) instead of an alert? 

I thought about it but didn't realize that's so annoying, let me reconsider :)